modules/apps/staging/staging-test/src/testIntegration/java/com/liferay/staging/test/StagingGetScopeGroupIdTest.java

```diff
@@ -64,27 +62,31 @@ public void setUp() throws Exception {
 		_mockHttpServletRequest.setAttribute(WebKeys.LAYOUT, stagingLayout);
 	}
 
-	@Ignore
 	@Test
-	public void testGetScopeGroupId() throws Exception {
-		_testGetScopeGroupId(
-			false, false, JournalContentPortletKeys.JOURNAL_CONTENT);
-		_testGetScopeGroupId(
-			false, true, JournalContentPortletKeys.JOURNAL_CONTENT);
-		_testGetScopeGroupId(
-			true, false, JournalContentPortletKeys.JOURNAL_CONTENT);
-		_testGetScopeGroupId(
-			true, true, JournalContentPortletKeys.JOURNAL_CONTENT);
-
+	public void testGetScopeGroupIdBlogs1() throws Exception {
 		_testGetScopeGroupId(false, false, BlogsPortletKeys.BLOGS);
+	}
+
+	@Test
+	public void testGetScopeGroupIdBlogs2() throws Exception {
 		_testGetScopeGroupId(false, true, BlogsPortletKeys.BLOGS);
+	}
+
+	@Test
+	public void testGetScopeGroupIdBlogs3() throws Exception {
 		_testGetScopeGroupId(true, false, BlogsPortletKeys.BLOGS);
+	}
+
+	@Test
+	public void testGetScopeGroupIdBlogs4() throws Exception {
 		_testGetScopeGroupId(true, true, BlogsPortletKeys.BLOGS);
 	}
 
 	@Override
 	protected String[] getNotStagedPortletIds() {
-		return new String[] {BlogsPortletKeys.BLOGS_ADMIN};
+		return new String[] {
+			BlogsPortletKeys.BLOGS_ADMIN, BlogsPortletKeys.BLOGS
+		};
 	}
```

Comment:
@DaniSzimko, `BlogsPortletKeys.BLOGS_ADMIN` seems to be unnecessary, please remove it.